Toolset/palettes/reverrordisplay.livecodescript

```diff
@@ -443,9 +442,9 @@ setProp cCurrentObject pWhat
    local tParsedData
    repeat for each line l in (line 1 to -2 of pWhat)
       if tScriptError then 
-         put "     " & tab & revIDELookupError("compilation", (item 1 of l)) after tParsedData
+         put "     " & tab & revIDELookupError("compilation", l) after tParsedData
       else 
-         put "Type" & tab & revIDELookupError("execution", (item 1 of l)) after tParsedData
+         put "Type" & tab & revIDELookupError("execution", l) after tParsedData
```

Comment:
One of these lines has 5 spaces followed by tab and the other has 4 characters followed by tab. Is that intentional?